app/build.gradle.kts

```diff
@@ -66,6 +66,12 @@ dependencies {
     implementation("com.google.dagger:hilt-android:2.46.1")
     kapt("com.google.dagger:hilt-compiler:2.46.1")
 
+    // Retrofit
+    implementation("com.google.code.gson:gson:2.10.1")
+    implementation("com.squareup.retrofit2:retrofit:2.9.0")
```

Comment:
One thing I like to have, is a variable where I store the lib version, and then reuse it

In this case, the "2.9.0" could be stored

Ex:

def retrofit_version = "2.9.0"


implementation("com.squareup.retrofit2:retrofit:$retrofit_version")
implementation("com.squareup.retrofit2:converter-gson:$retrofit_version)

Reply:
For single module apps I actually prefer putting the version inline, since it allows you to easily upgrade (I'm not sure if AS is nowadays capable of doing the Alt+Enter trick when putting it in a variable)

Reply:
Yes AS is still capable of the Alt+Enter trick